cvat-core/src/api-implementation.js

```diff
@@ -238,6 +238,7 @@
             // prettier-ignore
             const projects = projectsData.map((project) => {
                 if (filter.withoutTasks) {
+                    project.task_ids = project.tasks;
```

Comment:
If it is fetched with tasks, I would prefer intiailize this field also ``project.task_ids = project.tasks.map((task) => task.id);``
In this case ``Project.prototype.preview.implementation`` can be significantly simplified.